x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/analytics/moving_percentiles.yml

```diff
@@ -100,7 +99,44 @@ setup:
   - match: { aggregations.histo.buckets.2.moving_percentiles.values.0.value: 4.0 }
 
 ---
-"Basic Search HDR":
+tdest:
```

Comment:
I don't love the shorter names.  This one in particular, you need to know that the default algorithm for percentiles is t-digest already for it to make sense. I think a couple more words to spell this out would be good.